enlighten/Plugins/PluginController.py

```diff
@@ -244,7 +244,7 @@ def __init__(self, ctl,
 
         # bindings
         self.button_process.clicked.connect(self.button_process_callback)
-        self.cb_connected.clicked.connect(self.connected_callback)
+        self.cb_connected.stateChanged.connect(self.connected_callback)
```

Comment:
I think the checkbox stateChanged handler has a bug where some clicks won't invoke a message.

Something to the effect of: 
![image](https://github.com/WasatchPhotonics/ENLIGHTEN/assets/124081765/eb1ade35-be1d-4717-a218-5d902e4aebb5)

Details can be found in older commits on Enlighten related to watchdog and checkbox (if this is in fact the bug I'm thinking of)

Reply:
Good spot. It did work in my test across multiple clicks, whether through the GUI or the --plugin option.  However, it sounds like [others](https://stackoverflow.com/q/66697607) have had problems at least with older Qt versions.

I went ahead and changed it from stateChanged (which some say should only be used on tri-state checkboxes) to toggled, which seems the recommended signal for bools.  (I went away from clicked() because that definitely doesn't respond to setChecked()).  This still works in testing.